### ISSUE TITLE
[CI] DocsClientYamlTestSuiteIT {yaml=reference/cat/allocation/line_8} test failure

### ISSUE THREAD

**Issue by martijnvg** [elastic/elasticsearch#26685]
Reproduce line: `gradle :docs:integTestRunner -Dtests.seed=7285D296C382174B -Dtests.class=org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT -Dtests.method="test {yaml=reference/cat/allocation/line_8}" -Dtests.security.manager=true -Dtests.locale=ar-SA -Dtests.timezone=Europe/Minsk`

However this does not reproduce.

Actual failure:

```
FAILURE 0.17s | DocsClientYamlTestSuiteIT.test {yaml=reference/cat/allocation/line_8} <<< FAILURES!
   > Throwable #1: java.lang.AssertionError: Failure at [reference/cat/allocation:21]: field [$body] was expected to match the provided regex but didn't
   > Expected: shards \s+disk.indices \s+disk.used \s+disk.avail \s+disk.total \s+disk.percent \s+host      \s+ip        \s+node\s* \s+5         \s+\d+b    \s+\d+(\.\d+)?[tgmk]?b     \s+\d+(\.\d+)?[tgmk]?b    \s+\d+(\.\d+)?[tgmk]?b           \s+\d+ \s+127.0.0.1 \s+127.0.0.1 \s+.+\s*
   >      but: was "shards disk.indices disk.used disk.avail disk.total disk.percent host      ip        node\n     5        1.1kb     214gb    793.7gb   1007.8gb           21 127.0.0.1 127.0.0.1 node-0\n"
   > 	at __randomizedtesting.SeedInfo.seed([7285D296C382174B:FAD1ED4C6D7E7AB3]:0)
   > 	at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.executeSection(ESClientYamlSuiteTestCase.java:350)
   > 	at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.test(ESClientYamlSuiteTestCase.java:330)
   > 	at java.lang.Thread.run(Thread.java:748)
   > Caused by: java.lang.AssertionError: field [$body] was expected to match the provided regex but didn't
   > Expected: shards \s+disk.indices \s+disk.used \s+disk.avail \s+disk.total \s+disk.percent \s+host      \s+ip        \s+node\s* \s+5         \s+\d+b    \s+\d+(\.\d+)?[tgmk]?b     \s+\d+(\.\d+)?[tgmk]?b    \s+\d+(\.\d+)?[tgmk]?b           \s+\d+ \s+127.0.0.1 \s+127.0.0.1 \s+.+\s*
   >      but: was "shards disk.indices disk.used disk.avail disk.total disk.percent host      ip        node\n     5        1.1kb     214gb    793.7gb   1007.8gb           21 127.0.0.1 127.0.0.1 node-0\n"
   > 	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
   > 	at org.elasticsearch.test.rest.yaml.section.MatchAssertion.doAssert(MatchAssertion.java:67)
   > 	at org.elasticsearch.test.rest.yaml.section.Assertion.execute(Assertion.java:76)
   > 	at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.executeSection(ESClientYamlSuiteTestCase.java:346)
   > 	... 37 more
```

Build url: https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+master+intake/458/consoleText

**Comment by martijnvg** [comment]
Other instances of this failure: 
* https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.x+periodic/27/console
* https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.x+oracle-periodic/630/console

**Comment by tlrx** [comment]
Closed by c16c653c3e9ad07ddd90d2d78ef8337cc5294750
